src/app/auth/new-password/new-password.page.ts

```
@@ -1,5 +1,5 @@
 import { Component, OnInit } from '@angular/core';
```

ESLint

[error] 'Component' is defined but never used.

(@typescript-eslint/no-unused-vars)